warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/partition/MultiTableRangePartitioner.java

```diff
@@ -115,35 +117,38 @@ public static void setContext(TaskInputOutputContext<?,?,?,?> context) {
     }
     
     @Override
-    public void initializeJob(Job job) {
-        try {
-            Configuration conf = job.getConfiguration();
-            // only create the splits file if we haven't already created it, possibly for another table
-            if (null == NonShardedSplitsFile.findSplitsFile(job.getConfiguration(), job.getLocalCacheFiles(), isTrimmed())) {
-                URI splitsFileUri = createTheSplitsFile(conf);
-                job.addCacheFile(splitsFileUri);
-            }
-        } catch (Exception e) {
-            log.error(e);
-            throw new RuntimeException("Failed to initialize partitioner for job", e);
-        }
-    }
-    
-    private URI createTheSplitsFile(Configuration conf) throws IOException, URISyntaxException, TableNotFoundException, TableExistsException {
-        int reduceTasks = conf.getInt("splits.num.reduce", 1);
-        String[] tableNames = conf.get(TableConfigurationUtil.JOB_OUTPUT_TABLE_NAMES).split(",");
-        Path workDirPath = new Path(conf.get("ingest.work.dir.qualified"));
-        FileSystem outputFs = FileSystem.get(new URI(conf.get("output.fs.uri")), conf);
-        NonShardedSplitsFile.Writer writer = new NonShardedSplitsFile.Writer(conf, reduceTasks, workDirPath, outputFs, tableNames, isTrimmed());
-        writer.createFile(isTrimmed());
-        return writer.getUri();
-    }
-    
-    protected boolean isTrimmed() {
-        return true;
-    }
-    
-    @Override
+    // <<<<<<< HEAD
```

Comment:
Did you forget to resolve this?

Reply:
yep, this commented out code can be deleted now